server/deployment/base/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ resources:
 - treetracker-admin-api-service.yaml
```

Comment:
@arunbakt How do you deploy the service? I saw ppl add yaml in .github folder for deployment, how do you do it here?

Reply:
Yeah, the deployment folder seems to be used by the github workflow to deploy it to the k8 cluster.  I am still trying to understand the details on it.